ruby-push-notifications.gemspec

```diff
@@ -20,6 +16,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.add_dependency 'activesupport', '~> 4.2'
```

Comment:
Hmm, weird, there's nothing about `activesupport` in builder's dependencies: https://rubygems.org/gems/builder.

I think forcing the requirement here is a bad idea, since projects using rails 5 depend on `activesupport ~> 5.0`.

If it's for test purpose, we need to have a build matrix with different version of `activesupport` depending on the ruby version.
I did something like that in https://github.com/elhu/peastash.

If we actually need it at runtime and not just in specs, we need to:
* Figure out where it comes from
* See if we can't just leave the dependency there and let bundler sort it out